src/BaseballGameFunc.js

```diff
@@ -0,0 +1,76 @@
+const MissionUtils = require('@woowacourse/mission-utils');
+const Console = MissionUtils.Console;
+
+class BaseballGameFunc {
+  static inputControl = num => {
+    const regValidNumExp = /^[1-9]{3}$/;
+    const testValidationSet = new Set(num.split(''));
+    if (!regValidNumExp.test(num) || testValidationSet.size !== 3) {
+      throw '유효한 값이 아닙니다!';
+    }
+  };
+
+  static parser = num => {
```

Comment:
```suggestion
  static parser = (num) => {
```
이렇게 깜싸주시는 것이 이게 화살표 함수라는 것 더 쉽게 알아 볼 수 있을 것 같아요:)